src/super_gradients/examples/resnet_qat/resnet_qat_example.py

```diff
@@ -0,0 +1,44 @@
+"""
+QAT example for Resnet18
+
+The purpose of this example is to demonstrate the usage of QAT in super_gradients through phase callbacks.
```

Comment:
please explain that you are loading a model which was already trained

Reply:
Added https://github.com/Deci-AI/super-gradients/pull/232/commits/ceed91c396414059bd3ea5b12d7c195461fc42a3